campaignion_loqate/campaignion_loqate.info

```diff
@@ -5,3 +5,5 @@ core = 7.x
 dependencies[] = clientside_validation:clientside_validation
 dependencies[] = manual_direct_debit_uk
 dependencies[] = variable
+
+scripts[] = campaignion_loqate.min.js
```

Comment:
When the js is added in the info file, it doesn’t need `$form['#attached']['js'][] = drupal_get_path('module', 'campaignion_loqate') . '/campaignion_loqate.min.js';` anymore, does it?

Reply:
Yes, we need to add it globally now, because the `clientsideValidationAddCustomRules` event is only fired on the initial page load. I’ve removed the `#attach` line.